assets/global.js

```diff
@@ -553,6 +564,131 @@ class SliderComponent extends HTMLElement {
 
 customElements.define('slider-component', SliderComponent);
 
+class SlideshowComponent extends SliderComponent {
+  constructor() {
+    super();
+    this.sliderControlWrapper = this.querySelector('.slider-buttons');
+    this.bannerContents = this.slider.querySelectorAll('.banner__content');
```

Comment:
Would you mind having a:

```js
this.elements = {
 // all the variables that you are assigning to a querySelector
}
```
?

I think we're using this pattern in a few places, but it really helps with legibility. I'm often unsure what is a numeric / boolean variable and what is an element.

Reply:
Maybe we can take a look together/pair. I feel like some values are also used in `SliderComponent` and some are based on conditionals. So curious to see what would make the most sense.